gamestonk_terminal/stocks/options/options_controller.py

```diff
@@ -1260,6 +1262,80 @@ def call_parity(self, other_args: List[str]):
         )
         print("")
 
+    @try_except
+    def call_binom(self, other_args: List[str]):
+        """Process binom command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="parity",
+            description="Shows whether options are over or under valued",
+        )
+        parser.add_argument(
+            "-s",
+            "--strike",
+            type=float,
+            default=0,
+            dest="strike",
+            help="Strike price for optiom shown",
+        )
+        parser.add_argument(
+            "-p",
+            "--put",
+            action="store_true",
+            default=False,
+            dest="put",
+            help="Value a put instead of a call",
+        )
+        parser.add_argument(
+            "-e",
+            "--european",
+            action="store_true",
+            default=False,
+            dest="europe",
+            help="Value a European option instead of an American one",
+        )
+        parser.add_argument(
+            "--export",
+            action="store_true",
+            default=False,
+            dest="export",
+            help="Export an excel spreadsheet with binomial pricing data",
+        )
+        parser.add_argument(
+            "-P",
```

Comment:
Personally the capital P confuses me, so I would leave --plot only.  Just personal